### PR TITLE
jobs: added a job for putting a new snappy deb into the PPA.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
Daily for now. I'm working on the push trigger.

File: containers/jenkins-master/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM ubuntucore/jenkins-ubuntu
+
+  FROM ubuntucore/jenkins-ubuntu
```
Comment (fgimenez) [inline containers/jenkins-master/Dockerfile]: This spaces should be removed

**Comment by fgimenez** [comment]
One nitpick and :+1:

**Comment by elopio** [comment]
Done, and added dput as a slave dependency.

**Pull request merged**